install/installer/pkg/common/objects.go

```diff
@@ -40,17 +40,18 @@ func DefaultServiceAccount(component string) RenderFunc {
 }
 
 type ServicePort struct {
```

Comment:
This struct is now very close to the underlying `corev1.ServicePort`. We could consider using that instead as the extra struct isn't adding much anymore.